tests/fsm/test_regex.py

```diff
@@ -542,12 +542,14 @@ def convert_token_to_string(self, token):
         "�",
         "��",
         "�.",
-        "�..",
+        ".�",
+        ".�.",
         "▁�",
-        "▁▁�",
-        "▁�.",
-        "▁�.",
-        "▁▁�..",
```

Comment:
So multichars prefixes/suffixes of `�` doesn't need to be covered for these tokenizers?

Reply:
That's right. All cases "in the wild" include only a single prefix and/or suffix non-� character. I created these test cases, and I suppose I made them unnecessarily complex.

Reply:
Got it, alright, lets merge it!